packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -277,6 +285,25 @@ gmaps.CircleOptions _circleOptionsFromCircle(Circle circle) {
   return circleOptions;
 }
 
+gmaps_visualization.HeatmapLayerOptions _heatmapOptionsFromHeatmap(
+  Heatmap heatmap,
+) {
+  final heatmapOptions = gmaps_visualization.HeatmapLayerOptions()
```

Comment:
I'd definitely shorten `gmaps_visualization` here to `viz` or similar.

Reply:
Done